drivers/windows/R/resource.R

```diff
@@ -0,0 +1,13 @@
+## windows-specific cluster functions, called from hipercow.
+
+windows_cluster_info <- function(res) {
```

Comment:
```suggestion
windows_cluster_info <- function(config, path_root) {
```

then switch on `config$cluster`, erroring if it's not known. Ignore the `path_root ` arg